main.js

```diff
@@ -1,2 +1,14 @@
 'use strict'
-console.log('hello world!')
+
+const files = [
+  'track1.wav',
+  'track2.wav',
+  '.cache'
+]
+
+const listTrackFiles = () => {
+  return files.filter( file => file.endsWith('.wav') )
```

Comment:
I've seen a lot of people that struggles with the idea of the implicit params when using HoF.

I think it would be better to have the function inside the filter, outside, it its own variable, so people can understand that by using a function, it will receive each values implicitly

so 
```
const endsWithWav = file => file.endsWith('.wav')
return files.filter(endsWithWav );
```

My two cents.